playground/vite-react/src/App.css

```
@@ -1,42 +1,258 @@
-  transition: filter 300ms;
+* {
+  box-sizing: border-box;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+.app {
+  width: 100%;
+  min-height: 100vh;
+
+  background: var(--color-bg-light);
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)